cartesian_impedance_msgs/msg/CartesianDamping.msg

```diff
@@ -0,0 +1,11 @@
+# Damping parameters for the Cartesian Impedance
+# the values are divided into translational and rotational
+# each of the translational and rotational components is defined as Vector3
+# for more info on Vector3 messages please see the following link:
+# http://docs.ros.org/api/geometry_msgs/html/msg/Vector3.html
+
+
+# [N/m]
+geometry_msgs/Vector3 translational
+# [Nm/rad]
+geometry_msgs/Vector3 rotational
```

Comment:
In this context, what does rotational damping mean?  I assume it's dampening about the fixed frame axes?  Is there any confusion/issues like those associated with euler angles (i.e. order of application)?